point_cloud_transport/include/point_cloud_transport/subscriber_plugin.hpp

```diff
@@ -277,6 +351,17 @@ class POINT_CLOUD_TRANSPORT_PUBLIC SubscriberPlugin
     const Callback & callback,
     rmw_qos_profile_t custom_qos,
     rclcpp::SubscriptionOptions options) = 0;
```

Comment:
Provide implementation?
```suggestion
    rclcpp::SubscriptionOptions options)
  {
    subscribeImpl(
        node_interfaces,
        base_topic, callback,
        rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(custom_qos), custom_qos), options);
  }
```